.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,48 @@
+name: Feature request
+description: Suggest an idea for this project
+labels: [enhancement]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this issue! :hugs:
```

Comment:
You don't want to thank people for filling in some form. As explained on IRC, I'd rather encourage them for their interest in making NewPipe better. For example, "Thank you for making NewPipe better by reporting a bug." After such a welcoming sentence, you can, e.g., ask them to fill in as much information as they have, define some rules etc.